.pre-commit-config.yaml

```diff
@@ -11,20 +11,14 @@ repos:
     - id: check-added-large-files
  - repo: local
    hooks:
-    - id: solium
-      name: Solidity linter
-      language: node
-      entry: solium
-      files: '\.sol$'
-      args:
-      - --dir=solidity/contracts/
-      - --config=solidity/.soliumrc.json
-      additional_dependencies:
-      - ethlint@1.2.3
+    - id: lint-sol
+      name: 'lint solidity sol'
+      entry: /usr/bin/env bash -c "cd solidity && npm run lint:sol"
+      language: script
+      description: "Checks solidity code according to the package's linter configuration"
     - id: lint-js
       name: 'lint solidity js'
       entry: /usr/bin/env bash -c "cd solidity && npm run lint:js"
-      files: 'solidity\/.*\/\.js$'
```

Comment:
We should probably leave `files` property configured for both `lint-sol` and `lint-js`. It would execute linting only if there are changes to files matching a pattern.

Reply:
According to the [docs](https://pre-commit.com/) it looks like it is an optional property. I've tried committing both `.js` & `.sol` with bad linting and in both cases it stopped me informing about it. It will just execute whatever command we have in `entry` property.

Reply:
This property determines what files should be included in the check. One of the implications is that if the file doesn't match the pattern the check will be skipped. In our case, this is helpful as we don't want to run contracts linting verification when we're committing changes to the Go client, any other directory other than `solidity`, or to files that don't match extension that linting checks.

Reply:
`43873c4` ok, makes sense. But I think there was an error in that regex, cause it didn't check the JavaScript files when I tried to commit it. Only when I slightly changed it, it started to check `.js` files upon committing them.